golang/go.mod

```diff
@@ -1,188 +1,194 @@
 module github.com/asecurityteam/sdcli
 
-go 1.21
+go 1.22.1
```

Comment:
Do we want to upgrade this to match the Dockerfile version?

Reply:
oh... yes.  doing it.